commit 2482029bd30db1908c0f70591d313627bdb05a85
Author: John Levon <john.levon@joyent.com>
Date:   2019-08-22T16:27:43+00:00 (7 weeks ago)
    
    MANTA-4489 ops is under-sized in a lab deployment
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/config/services/ops/service.json.lab b/config/services/ops/service.json.lab
new file mode 100644
index 0000000..2a27ab7
--- /dev/null
+++ b/config/services/ops/service.json.lab
@@ -0,0 +1,5 @@
+{
+	"params": {
+		"ram": 4096
+	}
+}
